advanced/Scripts/chronometer.sh

```diff
@@ -194,27 +241,40 @@ get_sys_stats() {
     
     # Get DHCP stats, if feature is enabled
     if [[ "$DHCP_ACTIVE" == "true" ]]; then
-      ph_dhcp_eip="${DHCP_END##*.}"
       ph_dhcp_max=$(( ${DHCP_END##*.} - ${DHCP_START##*.} + 1 ))
     fi
     
-    # Get alt DNS server, or print total count of alt DNS servers
-    if [[ -z "${PIHOLE_DNS_3}" ]]; then
-      ph_alts="${PIHOLE_DNS_2}"
-    else
-      dns_count="0"
-      [[ -n "${PIHOLE_DNS_2}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_3}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_4}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_5}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_6}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_7}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_8}" ]] && dns_count=$((dns_count+1))
-      [[ -n "${PIHOLE_DNS_9}" ]] && dns_count="$dns_count+"
-      ph_alts="${dns_count} others"
-    fi
+    # Get DNS server count
+    dns_count="0"
+    [[ -n "${PIHOLE_DNS_1}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_2}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_3}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_4}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_5}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_6}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_7}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_8}" ]] && dns_count=$((dns_count+1))
+    [[ -n "${PIHOLE_DNS_9}" ]] && dns_count="$dns_count+"
   fi
   
+  scr_size=( $(stty size 2>/dev/null || echo 24 80) )
+  scr_lines="${scr_size[0]}"
+  scr_cols="${scr_size[1]}"
+  
+  if [[ "$scr_cols" -ge 58 ]]; then
+    chrono_type="large"
+  elif [[ "$scr_cols" -gt 40 ]]; then
+    chrono_type="medium"
+  else
+    chrono_type="small"
+  fi
+  
+  # Determine max length of divider
+  scr_line_len=$(( ${scr_size[1]} - 2 ))
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2004

The $ is unavoidable for special variables like $1 vs 1, $# vs #. It's also required when adding modifiers to parameters expansions, like ${#var} or ${var%-}. ShellCheck does not warn about these cases.

Since Codacy is complaining, I can just assign `${scr_size[1]}` to its own variable, and perform arithmetic against that.